RiotSwiftUI/Modules/Spaces/SpaceSettings/Coordinator/SpaceSettingsModalCoordinator.swift

```diff
@@ -122,64 +122,69 @@ final class SpaceSettingsModalCoordinator: Coordinator {
     }
     
     private func exploreRooms(ofSpaceWithId spaceId: String) {
-        let coordinator = ExploreRoomCoordinator(session: parameters.session, spaceId: spaceId)
-        coordinator.delegate = self
-        add(childCoordinator: coordinator)
-        coordinator.start()
-        self.navigationRouter.present(coordinator.toPresentable(), animated: true)
+        // Tchap: Disable Spaces
```

Comment:
Could we ignore this file in project ?

Reply:
Please revert those changes.

Reply:
I cannot change this, because this change is mandatory to build the project, and this file cannot be removed from the target. There are a lots of dependencies with the Spaces feature everywhere inside the code.